docs/distributions/manjaro/faq.md

```diff
@@ -1,30 +1,60 @@
+# Migrating to new kernel
+
+Run this in your terminal:
+
+```sh
+cat << EOF | sudo tee -a /etc/pacman.conf
+[manjaro-mact2]
+SigLevel = Never
+Server = https://mirror.funami.tech/$repo/os/$arch
```

Comment:
Have you checked that the $repo bits are written to the file? I don't remember if they get interpreted and turned to "", so you might need a \$ if it doesn't work.

Reply:
Yeah they turn to "". The same issue was faced my me while writing the touchbar script.